cognite/neat/_graph/transformers/_value_type.py

```diff
@@ -64,3 +71,73 @@ def transform(self, graph: Graph) -> None:
                     graph.remove((s, property_uri, o))
                     new_property = URIRef(f"{property_uri}_{remove_namespace_from_uri(value_type_uri)}")
                     graph.add((s, new_property, o))
+
+
+class ConvertLiteral(BaseTransformer):
+    description: str = "ConvertLiteral is a transformer that converts a literal value."
+    _use_only_once: bool = False
+    _need_changes = frozenset({})
+
+    _count_by_properties = """SELECT (COUNT(?property) AS ?propertyCount)
+    WHERE {{
+      ?instance a <{subject_type}> .
+      ?instance <{subject_predicate}> ?property
+    }}"""
+
+    _properties = """SELECT ?instance ?property
+    WHERE {{
+      ?instance a <{subject_type}> .
+      ?instance <{subject_predicate}> ?property
+    }}"""
+
+    def __init__(
+        self,
+        subject_type: URIRef,
+        subject_predicate: URIRef,
+        conversion: Callable[[Any], Any],
+    ) -> None:
+        self.subject_type = subject_type
+        self.subject_predicate = subject_predicate
+        self.conversion = conversion
```

Comment:
you should default conversion to `string_to_ideal_type` which is in `cognite.neat._utils.auxliliar`

Reply:
As I understand it, this will not solve the motivating use case for this transformer. In this case, we are importing from classic TimeSeries which has isString as a boolean. While, in CogniteTimeSeries we need this to be an enum with value 'string' and 'numeric'.